core/test/src/test_membank.cpp

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018 Robin Linden <dev@robinlinden.eu>
+
+#include "../../src/membank.h"
```

Comment:
This does not look so good. How shall we handle this @robinlinden ?

Reply:
I figured testing the external interface should be enough. Aren't the membanks tested via the mmu tests? If that's not enough, we'll probably have to this about this for a bit. Exposing internal headers for the sake of unit tests seems a bit messy.

Reply:
You are right, it should be covered by the mmu tests. I'll remove this file.